README.md

```diff
@@ -161,6 +161,7 @@ This section is a great place to start if you want to get into improving OpenStr
 * [OsmAPP](https://osmapp.org) - A browsable map showing data and offering simple POI-editing capabilities. ([Source Code](https://github.com/zbycz/osmapp))
 * [Tracesmap](https://tracesmap.com/) - A map allowing for planning routes on OsmCarto and OpenTopoMap-derived retina maps.
 * [Visit Sights](https://visitsights.com/) - Suggestions for self-guided sightseeing tours by foot and overview of individual sights all around the world.
+* [Defikarte.ch - Die Defikarte der Schweiz](https://www.defikarte.ch) - A Map that shows all available defibrillators in Switzerland and Liechtenstein, also used by emergency dispatch centers and rescue services.
```

Comment:
Please remove the tagline here. I feel listing the tagline here would be a bit confusing due to it being in German, and unfair because we're not listing the taglines of any other projects either. Also I feel like a small notice as to the site only being available in German is needed.

```suggestion
* [Defikarte.ch](https://www.defikarte.ch) - A Map that shows all available defibrillators in Switzerland and Liechtenstein, also used by emergency dispatch centers and rescue services. (ℹ️ German only)
```